genologics/xml_comparison.py

```diff
@@ -0,0 +1,31 @@
+import re
+import xml.etree.ElementTree as ET
+
+
+class ComparableXml(object):
```

Comment:
FYI: There is a library for comparing lxml trees here: https://pypi.org/project/xmldiff/ based on a paper: http://ilpubs.stanford.edu:8090/115/1/1995-46.pdf

Might be worth it to include it.